allennlp/semparse/domain_languages/wikitables_language.py

```diff
@@ -310,12 +350,42 @@ def all_rows(self) -> List[Row]:
         return self.table_data
 
     @predicate
-    def select(self, rows: List[Row], column: Column) -> List[str]:
+    def select_string(self, rows: List[Row], column: Column) -> List[str]:
```

Comment:
Shouldn't this take a `StringColumn` now?  Then you won't need the `assert`, also.

Reply:
Yes, thanks!